src/forge/ConfidentialStoreConnector.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.8;
+
+import "../suavelib/Suave.sol";
+
+contract ConfidentialStoreConnector {
+    fallback() external {
+        address confidentialStoreAddr = 0x0101010101010101010101010101010101010101;
+
+        address addr = address(this);
+        bytes4 sig;
+
+        // You can use 'forge selectors list' to retrieve the function signatures
+        if (addr == Suave.CONFIDENTIAL_STORE) {
+            sig = 0xd22a3b0b;
```

Comment:
Would `ConfidentialStore.confidentialStore.selector` work?

Reply:
Works! I remember trying combinations of that without success but it works with your code.